tests/test_interface_svi.rb

```diff
@@ -35,11 +56,17 @@ def assert_result(expected_result, err_msg, &block)
     end
   end
 
+  def skip_autostate_test
```

Comment:
nit: Rob's been using the question mark syntax on these kinds of platform check methods, ie. `skip_autostate_test?`